iib/workers/tasks/build.py

```diff
@@ -33,7 +33,6 @@
     skopeo_inspect,
 )
 
-
```

Comment:
This file has a bunch of whitespace lines removed that have nothing to do with the PR. Could you please remove this deletion from the commit?